src/components/ContactUs.jsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+
+const ContactUs = ({ data }) => {
+  const classes = useStyles();
+
+  return (
+    <div data-cy="contact-us" className={classes.container} style={styles.container}>
+      <Typography
+        data-cy="contact-us-header"
+        variant="h3"
+        component="h2"
+        className={classes.heading}
+      >
+        Contact Us
+      </Typography>
+      <Typography
+        data-cy="contact-us-email"
+        variant="body1"
+        component="p"
+        className={classes.text}
+      >
+        Email: {data ? data.email : `Sorry can't reach the server please try again later` }
+      </Typography>
+      <Typography
+        data-cy="contact-us-phone"
+        variant={"body1"}
+        component="p"
+        className={classes.text}
+      >
+        Phone: {data ? data.phone : `Sorry can't reach the server please try again later` }
+      </Typography>
+    </div>
+  );
+};
+
+export default ContactUs;
+
+const useStyles = makeStyles((theme) => ({
+  container: {
+    [theme.breakpoints.up("xs")]: {
+      maxWidth: "600px",
+      marginTop: "10%",
+    },
+    [theme.breakpoints.up("md")]: {
+      marginTop: "15%",
+      marginLeft: "20%",
+    },
+  },
+
+  heading: {
+    [theme.breakpoints.up("xs")]: {
+      fontSize: "28px",
+    },
+    [theme.breakpoints.up("md")]: {
+      fontSize: "46px",
+    },
+  },
+
+  text: {
+    [theme.breakpoints.up("xs")]: {
+      fontSize: "16px",
+      marginTop: "16px",
+      wordWrap: "break-word",
+    },
+    [theme.breakpoints.up("md")]: {
+      fontSize: "18px",
+      marginTop: "18px",
+    },
+  },
+}));
```

Comment:
In Mui documentation they put useStyles inside function component right after declarations of state, variables, etc. It wont change functionality but lets do the same way, following conventions.

Reply:
But useStyles is the way to go when doing responsive stuff. I now wonder if we can transfer all font related stuff into the theme, so we don't need to repeat all that in every component? Also they use this rem units for fonts, does that have anything to do with responsiveness?

Reply:
yeah i just placed it bellow as we do with ```styles``` so it don't get so messy before the "code"
that is how we did with style as well i believe (it was on the top in docs but dose't matter where it is so place under export for readability) I don't really mind so can put it back up for now and we can see how it feels and maybe move it back in the future.